code/modules/hydroponics/grown.dm

```diff
@@ -147,6 +147,16 @@
 	if(seed) 
 		seed.thrown_at(src,hit_atom)
 
+var/list/_wood_materials = list(
```

Comment:
can't this just be istype(/decl/material/solid/wood)?

Reply:
You'd have to iterate the entire chems list and it would include stuff like holographic wood.